_data/CONTRIBUTORS.yaml

```diff
@@ -540,3 +540,5 @@ Dirk Winkelhardt:
     orcid: 0000-0001-8770-2221
     email: dirk.winkelhardt@rub.de
     affiliation: Ruhr-Universität Bochum, Medzinisches Proteom-Center & ELIXIR Germany Forschungszentrum Jülich
+Carolina Simón Guerrero:
```

Comment:
If possible, add also the other fields, especially the ORCID.

Reply:
Done! :smile: